src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";

```

Comment:
The `DB_URL` is incomplete. It should specify the database name, and potentially other parameters such as character encoding and server timezone. For example: `jdbc:mysql://localhost:3306/myDatabase?useUnicode=true&serverTimezone=UTC`.